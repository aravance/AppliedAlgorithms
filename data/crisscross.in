69 72 6 82
42 45 87 10
17 85 2 41
83 85 12 68
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
999999 999999 999999 999999
999999 999999 999999 999999
999999 999999 999999 999999
999999 999999 999999 999999
999999 1 1 1
1 999999 1 1
1 1 999999 1
1 1 1 999999
1 1 1 999999
1 1 999999 1
1 999999 1 1
999999 1 1 1
69 42 17 83
72 45 85 85
6 87 2 12
82 10 41 68
1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4
5 6 7 8
9 10 11 12
13 14 15 16
0 0 0 0
