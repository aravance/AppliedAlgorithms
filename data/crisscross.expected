261
3
2333331
3999995
3999995
261
11
39
